pytrainer/test/lib/test_date.py

```diff
@@ -65,7 +65,23 @@ def test_getDate_should_raise_ValueError_if_day_is_before_first(self):
         mock_calendar.configure_mock(**attrs)
         with self.assertRaises(ValueError):
             Date(mock_calendar).getDate()
-
+            
+    def test_getDate_should_return_valid_date_if_date_is_valid(self):
+        mock_calendar = Mock()
+        start_date = datetime.date(2019, 1, 1)
+        # list of every day in 2019
+        date_list = [start_date + datetime.timedelta(days=x) for x in range(0, 365)]
+        end_date = datetime.date(2019, 12, 31)
+        # test every day in the list
+        for date in date_list:
+           with self.subTest(date=date):
```

Comment:
Sadly subTest isn't supported on Python 2.7 (which we still need to support at least for this release). Can you remove that, and rebase on top of current master (which fixes issues with new versions of the mock library)?